lib/3scale/backend/stats/partition_eraser_job.rb

```diff
@@ -11,13 +11,12 @@ class << self
           include StorageHelpers
           include Configurable
 
-          def perform_logged(_enqueue_time, service_id, applications, metrics, users,
+          def perform_logged(_enqueue_time, service_id, applications, metrics,
```

Comment:
This is dangerous. How are we going to run pending jobs from an upgrade?

We should make the jobs' signatures callable with newer and older enqueued jobs for a while, otherwise this will break when upgrading from a previous version.

Reply:
There are no partition_eraser jobs running. We disabled them for performance reasons.

Reply:
No any other job affected? IIRC there is at least another one `perform_logged`. If also not enabled then ok to go.

Reply:
It should be fine. `ReportJob` and `NotifyJob` receive transactions. There isn't a `user_id` param.